pkg/machines/actions/provider-actions.es6

```diff
@@ -184,6 +186,26 @@ export function setVCPUSettings(vm, max, count, sockets, threads, cores) {
     });
 }
 
+export function setMem(vm, memory) {
+    return virt(SET_MEMORY, {
+        name: vm.name,
+        id: vm.id,
+        connectionName: vm.connectionName,
+        memory,
+        isRunning: vm.state == 'running'
+    });
+}
+
+export function setMaxMem(vm, maxMemory) {
```

Comment:
Please in all functions names use full names, same with the actions types names, aka setMemory, setMaxMemory.